cranelift/filetests/src/test_zkasm.rs

```diff
@@ -237,6 +240,62 @@ mod tests {
         program.join("\n")
     }
 
+    fn run_wat_file(path: &Path) -> Result<(), Box<dyn std::error::Error>> {
+        let engine = Engine::default();
+        let wat = std::fs::read_to_string(path).unwrap();
+        let pat = r#"(import "env" "assert_eq" (func $assert_eq (param i32) (param i32)))"#;
+        let assert_type = if wat.contains(pat) { 32 } else { 64 };
+        let binary = wat::parse_file(path)?;
+        let module = Module::new(&engine, &binary)?;
+        if assert_type == 64 {
+            // 4 a magic number which we must provide for Store initialization, but not important in our case
+            // a searched ways to use some default numbers, but unsuccesfully, so took a number same as in
+            // example in docs
+            // TODO: find a way to do it without magic numbers
+            let mut store = Store::new(&engine, 4);
```

Comment:
A link to these examples would be helpful.

nit: Would it be possible to use `()` instead of 4? Then `Store<()>` instead of `Store<4>` might indicate more clearly that we’re not using host data with the store.